src/store/block.js

```diff
@@ -85,7 +85,7 @@ export default {
 		resolutionStatement: state => state.blockReceipts?.data?.resolutionStatements || [],
 		currentBlockHeight: state => state.currentBlockHeight,
 
-		infoText: (s, g, rs, rootGetters) => 'Chain height: ' + rootGetters['chain/getBlockHeight']
+		infoText: (s, g, rs, rootGetters) => 'Chain height: ' + rootGetters['chain/getChainInfo'].currentHeight
```

Comment:
This will handle empty data:
```	
infoText: (s, g, rs, rootGetters) => 
   translate('chainHeight') +': ' + 
   (rootGetters['chain/getChainInfo'] && rootGetters['chain/getChainInfo'].currentHeight 
      ? rootGetters['chain/getChainInfo'].currentHeight` 
      : translate('noData'))
```

Reply:
The default data already define in `chain/getChainInfo` state.